rootfs/etc/nginx/lua/balancer/static_balancers.lua

```diff
@@ -0,0 +1,83 @@
+local util = require('util')
+local split = require('util.split')
+local ngx_upstream = require("ngx.upstream")
+local implementations = require('balancer.implementations')
+
+local static_backends = {}
+local static_balancers = {}
+
+local DEFAULT_LB_ALG = "ewma"
+
+local _M = {}
+
+local function marshal_endpoint(endpoint)
+    if (not endpoint.address) or (not endpoint.port) then
+        if endpoint.addr then
+            local addr, err = split.parse_addr(endpoint.addr)
+            if err then
+                return nil, err
+            end
+
+            endpoint.address = addr.host
+            endpoint.port = addr.port
+            endpoint.addr = nil
+
+            return endpoint, nil
+        end
+    end
+    return nil, "error in grabbing address & port" 
+end
+
+local function create_static_backend(upstream_name)
```

Comment:
the module name already sets the expectation that operations in this module are about static backends, therefore I'd just call it `create_backend`. If we would be generating something other than static then it'd be valuable to include that in the name (i.e `create_non_static_backend`).